README.md

```diff
@@ -1,10 +1,15 @@
-# webots-cloud-simulation-demos
+# [webots.cloud](https://webots.cloud) Simulation Demos
+
+## Introduction
+
+This repository provide a series of simple Webots simulations which can be run on the web in [webots.cloud](https://webots.cloud).
```

Comment:
```suggestion
This repository provide a series of simple Webots simulations which can be run on the web on [webots.cloud](https://webots.cloud).
```

I am not sure about this

Reply:
I assumed it was similar to saying "running in the cloud" which is more common that "running on the cloud".

Reply:
https://medium.com/click-travel/running-on-the-cloud-or-in-the-cloud-which-one-are-you-doing-69d2ff1196ed